types/rails__ujs/utils/dom.d.ts

```diff
@@ -0,0 +1,11 @@
+export interface SelectorObject {
+    selector: string;
+    exclude?: string;
```

Comment:
It seems that ʻexclude` is required when giving an object. If you give an object without ʻexclude`, it doesn't seem to work.

https://github.com/rails/rails/blob/v6.0.3.4/actionview/app/assets/javascripts/rails-ujs/utils/dom.coffee#L12-L14

Reply:
I thought it's optional because there is `if selector.exclude?` ( `if (selector.exclude != null)` when compiled to JS ). But now I found that it is actually checking the `selector` is a string, thanks.